app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :Comments
+  has_many :likes_counter
+
+  after_save :updates_posts_counter
+
+  def updates_posts_counter
+    user.increment!(:PostsCounter)
+  end
+
+  def recent_comments
+    comments.order(created_at: :desc).last(5)
+  end
```

Comment:
- [ ] with the `.last` method you're getting the last 5 items in the array which in this case are the 5 oldest comments, you should however try to get the 5 most recent comments.

Reply:
Thank so much and i will do just now